project/src/main/java/com/google/sps/utilities/SpannerTasks.java

```diff
@@ -75,59 +88,83 @@ public static void insertorUpdateEvent(Event event) {
     SpannerClient.getDatabaseClient().write(mutations);
   }
 
-  /** Returns List of Event Ids from DB
+  /**
+   * Returns List of Event Ids from DB
    *
    * @param eventId List of IDs of event to be returned
-  */
+   */
   public static Set<Event> getEventsFromIds(List<String> eventIds) {
-      Set<Event> ids = new HashSet<Event>();
-      for (String eventId: eventIds) {
-        Optional<Event> event = getEventById(eventId);
-        if (event.isPresent()) {
-          ids.add(event.get());
-        }
-        
+    Set<Event> ids = new HashSet<Event>();
+    for (String eventId : eventIds) {
+      Optional<Event> event = getEventById(eventId);
+      if (event.isPresent()) {
+        ids.add(event.get());
       }
-      return ids;
+    }
+    return ids;
   }
 
-  /** Returns Event by ID from DB
-   * 
+  /**
+   * Returns Event by ID from DB
+   *
    * @param eventId ID of event to be returned
-  */
+   */
   public static Optional<Event> getEventById(String eventId) {
     ResultSet resultSet =
         SpannerClient.getDatabaseClient()
             .singleUse()
             .executeQuery(
                 Statement.of(
                     String.format(
-                        "SELECT Name, Description, Labels, Location, Date, Time, Host,"
+                        "SELECT EventId, Name, Description, Labels, Location, Date, Time, Host,"
                             + " Opportunities, Attendees FROM %s WHERE EventID='%s'",
                         EVENT_TABLE, eventId)));
-    
+
     /** If ID does not exist */
     if (!resultSet.next()) {
       return Optional.empty();
     }
+    return Optional.of(shallowCreateEventFromDatabaseResult(resultSet));
+  }
 
-    // TO DO: replace with host from db, after PR #43 pushed
-    String NAME = "Bob Smith";
-    String EMAIL = "bobsmith@example.com";
-    User host = new User.Builder(NAME, EMAIL).build();
-    /* remove Hardcoded date */
-    Date date = Date.fromYearMonthDay(2016, 9, 15);
-    return Optional.of(
-        new Event.Builder(
-                /* name = */ resultSet.getString(0),
-               /* description = */ resultSet.getString(1),
-                /* labels = */ new HashSet<String>(resultSet.getStringList(2)),
-                /* location = */ resultSet.getString(3),
-                /* date = */ Date.parseDate(resultSet.getString(4)),
-                /* time= */ resultSet.getString(5),
-                /* host = */ host)
-            .build());
-    // TO DO: set volunteer opportunities, attendees by Querying those by ID, wait for PR 43, 44
+  /**
+   * Returns all events stored in DB
+   *
+   * @return Events with a shallow version of its host (no events attached to Users)
+   */
```

Comment:
nit: add full description in here Returns all events stored in DB {....} I think information can be repeated in return section but it should be in the description as well.

Reply:
Done